mobile/evidence/ios/Leaf/Sources/Leaf/Theme/LeafTheme.swift

```diff
@@ -2,46 +2,28 @@ import SwiftUI
 
 public struct LeafTheme {
     public let color: Color
-    public let font: Font
     public let size: Size
 }
 
 extension LeafTheme {
     public struct Color {
-        public let brand: BrandColor
-        public let content: ContentColor
-        public let tag: TagColor
+        public let system: SystemColors
+        public let custom: CustomThemeColors
     }
-    
-    public struct BrandColor {
-        public let primary: SwiftUI.Color
-        public let secondary: SwiftUI.Color
-        public let tertiary: SwiftUI.Color
-    }
-    
-    public struct ContentColor {
+
+    public struct SystemColors {
```

Comment:
Think this name is a little bit misleading since it defines custom colors for our domain, but when reading in code it seems like is actually something provided by the operational system.